Doc/reference/datamodel.rst

```diff
@@ -1255,6 +1255,41 @@ The :meth:`!co_positions` method
       :option:`-X` ``no_debug_ranges`` command line flag or the :envvar:`PYTHONNODEBUGRANGES`
       environment variable can be used.
 
+.. method:: codeobject.co_lines()
+
+   Returns an iterator that yields information about successive ranges of
+   :term:`bytecode`\s. Each item yielded is a ``(start, end, lineno)``
+   :class:`tuple`:
+
+   * ``start`` (an :class:`int`) represents the offset (inclusive) of the start
+     of the :term:`bytecode` range;
+   * ``end`` (an :class:`int`) represents the offset (inclusive) of the end of
+     the :term:`bytecode` range;
+   * ``lineno`` is an :class:`int` representing the line number of the
+     :term:`bytecode` range, or ``None`` if the bytecodes in the given range
+     have no line number.
+
+   The items yielded generated will have the following properties:
+
+   * The first range yielded will have a ``start`` of 0;
```

Comment:
To match the others in this list?
```suggestion
   * The first range yielded will have a ``start`` of 0.
```

I'd also end with fullstops or no punctuation in the preceding list, rather than semicolons, but up to you.

Reply:
removed the semicolons in ba2c504 :)